src/webgpu/api/validation/capability_checks/limits/maxBindGroups.spec.ts

```diff
@@ -39,47 +38,28 @@ g.test('createPipelineLayout,at_over')
     );
   });
 
-g.test('createPipeline,at_over')
-  .desc(`Test using createRenderPipeline and createComputePipeline at and over ${limit} limit`)
-  .params(kMaximumLimitBaseParams.combine('createPipelineType', kCreatePipelineTypes))
-  .fn(async t => {
-    const { limitTest, testValueName, createPipelineType } = t.params;
-
-    await t.testDeviceWithRequestedMaximumLimits(
-      limitTest,
-      testValueName,
-      async ({ device, testValue, shouldError }) => {
-        const lastIndex = testValue - 1;
-
-        const code = t.getGroupIndexWGSLForPipelineType(createPipelineType, lastIndex);
-        const module = device.createShaderModule({ code });
-
-        await t.expectValidationError(() => {
-          t.createPipeline(createPipelineType, module);
-        }, shouldError);
-      }
-    );
-  });
-
-g.test('createPipelineAsync,at_over')
+g.test('createPipeline,async,at_over')
```

Comment:
nit. Maybe we don't even need the word `async` here (and in the other tests). I think that the parameter makes it pretty obvious?

Reply:
Does it? If I saw a test called `createPipeline` would assume it does not test async.

Reply:
Ah, I see your point since I realized that the `async` param will be hidden in the subcases. What if we promoted the `async` param to come before `kMaximumLimitBaseParams` that way it can be in the explicit case name? Then in the UI viewer and in tests names we would see something like: `createPipeline,at_over:async=true;*` and `createPipeline,at_over:async=false;*`.

Reply:
I could do that ... my point was it seems reasonable to search just the names. For example searching for `g.test\('[^']*createPipeline[^']*'`

<img width="593" alt="Screenshot 2023-03-10 at 15 13 50" src="https://user-images.githubusercontent.com/234804/224445345-fa0a332d-3b83-42a4-a3dd-e68d66acd3ac.png">

or `g.test\('[^']*create[\w]*Pipeline[^']*'`

<img width="598" alt="Screenshot 2023-03-10 at 15 17 17" src="https://user-images.githubusercontent.com/234804/224445781-32db4241-02be-49b7-ba47-e5982b1bfe71.png">

to find tests that test `createXXXPipelineAsync`. If I didn't see "async" I wouldn't know where to look.

Surprised that this isn't useful though as it didn't find any other tests :P

Reply:
Hmm yea, I guess I'm not too set on either way, but searching through the codebase I also see other cases:

https://github.com/gpuweb/cts/blob/main/src/webgpu/api/validation/compute_pipeline.spec.ts
https://github.com/gpuweb/cts/blob/main/src/webgpu/api/validation/render_pipeline/fragment_state.spec.ts

where we have a param named `isAsync` instead of `async`. And the test name themselves don't both with saying async in them. Maybe it is good enough to include a mention of it in the description?

Reply:
done